### ISSUE TITLE
Generate README, man pages, and docs

### ISSUE THREAD

**Issue by BobWall23** [IronCoreLabs/ironssh#8]
Currently, ironssh Makefile doesn't build and docs or man pages. Should add man pages for ironsftp and ironscp, fix Makefile to build.

**Comment by zmre** [comment]
Note: currently have (or will soon check in) this Makefile snippet:

```
install-iron-files:
    $(srcdir)/mkinstalldirs $(DESTDIR)$(bindir)
    $(srcdir)/mkinstalldirs $(DESTDIR)$(mandir)
    $(srcdir)/mkinstalldirs $(DESTDIR)$(mandir)/$(mansubdir)1
    $(INSTALL) -m 0755 $(STRIP_OPT) ironsftp$(EXEEXT) $(DESTDIR)$(bindir)/ironsftp$(EXEEXT)
#   $(INSTALL) -m 644 ironsftp.1.out $(DESTDIR)$(mandir)/$(mansubdir)1/ironsftp.1
```

When the `ironsftp.1` man page is ready, please uncomment the line for installing it.

**Comment by zmre** [comment]
Actually, when the docs are ready to go, we'll need to update more than the Makefile. Will need to make sure the docs get included in the various platform packages as well.

**Comment by BobWall23** [comment]
Closed by PR #40
